threefoldtech/tfgrid-sdk-ts
# Dao proposal share button

### Changes

- added a share button to the dao proposal card
- when pressed, a link is copied to the clipboard
- when link is entered in browser, it opens up dao page with shared proposal card highlighted 

### Related Issues

#3066

### Tested Scenarios

- press share button on any executable/active proposal 
- observe copied to clipboard success toast appear 
- enter linked copied in a new browser tab
- observe shared proposal card highlighted based on selected app theme


### Checklist

https://www.loom.com/share/97f1e9fffe654007a04a916d62468371?sid=1877c39c-f354-451c-b96e-e2a13184b5c0
